Detectors/TRD/workflow/src/TRDGlobalTrackingSpec.cxx

```diff
@@ -835,21 +839,32 @@ DataProcessorSpec getTRDGlobalTrackingSpec(bool useMC, GTrackID::mask_t src, boo
 
   // Request PID policy data
   if (withPID) {
+    // request policy
     switch (policy) {
       case PIDPolicy::LQ1D:
-        // inputs.emplace_back("LQ1D", "TRD", "LQ1D", 0, Lifetime::Condition, ccdbParamSpec("TRD/ppPID/LQ1D"));
+        inputs.emplace_back("lq1dlut", "TRD", "LQ1D", 0, Lifetime::Condition, ccdbParamSpec("TRD_test/PID/LQ1D"));
         break;
       case PIDPolicy::LQ3D:
-        // inputs.emplace_back("LQ3D", "TRD", "LQ3D", 0, Lifetime::Condition, ccdbParamSpec("TRD/ppPID/LQ3D"));
+        inputs.emplace_back("lq3dlut", "TRD", "LQ3D", 0, Lifetime::Condition, ccdbParamSpec("TRD_test/PID/LQ3D"));
         break;
+#ifdef TRDPID_WITH_ONNX
```

Comment:
would there be any harm here in case one of these policies is loaded without ONNX present? Probably not, right, and then you would not need this `#ifdef` here

Reply:
Since I mask them out in the enum with the #ifdef, I also have to put it here. Otherwise the case would not be there